src/components/project/ProjectMetaData.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Box } from "@material-ui/core";
+import PlaceIcon from "@material-ui/icons/Place";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  orgLogo: {
+    height: "0.9rem",
+    marginBottom: -2
+  },
+  cardIconBox: {
+    width: 40,
+    display: "inline-block"
+  },
+  cardIcon: {
+    verticalAlign: "bottom",
+    marginBottom: -2,
+    marginTop: 2
+  }
+});
+
+export default function ProjectMetaData({ project }) {
```

Comment:
I'm fine to have this in a separate component, but it seems like it's really only useful in the `ProjectPreview` component. Why a separate component/file?

Reply:
I thought it makes it a bit clearer. If you think it makes more sense to not create new files for components, that are only used once, I will move it back to the main file.